File: src/easy/SmallerNumbersThanCurrent.swift
func smallerNumbersThanCurrent(_ nums: [Int]) -> [Int] {
  var result = Array(repeating: 0, count: nums.count)
  for i in 0..<nums.count {
    for j in 0..<nums.count {
      if nums[i] > nums[j] {
        result[i] += 1
      }
    }
  }
  return result
}

File: src/main/java/easy/SmallerNumbersThanCurrent.java
package easy;

public class SmallerNumbersThanCurrent {
  public int[] solve(int[] nums) {
    int[] result = new int[nums.length];
    for (int i = 0; i < nums.length; i++) {
      for (int j = 0; j < nums.length; j++) {
        if (nums[i] > nums[j]) {
          result[i]++;
        }
      }
    }
    return result;
  }
}
